Added: alt attribute to few images (#19842)
mdn/content: files/en-us/learn/server-side/express_nodejs/forms/delete_author_form/index.md

If the author has no books, you'll be presented with a page like this.
After pressing delete, the server will delete the author and redirect to the author list.

> ![The Delete Author section of the Local library application of an author who does not have any books. The left column has a vertical navigation bar. The right section contains the author details with a heading that contains the Author's name followed by the life dates of the author. There is a button labelled 'Delete' at the bottom.](locallibary_express_author_delete_nobooks.png)

Reviewer: ```suggestion
![The Delete Author section of the Local library application of an author who does not have any books. The left column has a vertical navigation bar. The right section contains the author's name and life dates. There is the question "Do you really want to delete this author" with a button labeled 'Delete'.](locallibary_express_author_delete_nobooks.png)
```

If the author does have books, then you'll be presented with a view like the following.
You can then delete the books from their detail pages (once that code is implemented!).

> ![The Delete Author section of the Local library application of an author who does have books under his name. The left column has a vertical navigation bar. The right section contains the author details with a heading that contains the Author's name followed by the life dates of the author. There is a statement that reads "Delete the following books before attempting to delete this author" and the books written by the author are listed after that.](locallibary_express_author_delete_withbooks.png)

Reviewer: ```suggestion
![The Delete Author section of the Local library application of an author who does have books under his name. The section contains the author's name and life dates of the author. There is a statement that reads "Delete the following books before attempting to delete this author" followed by the author's books. The list includes the titles of each book, as links, followed by a brief description in plain text.](locallibary_express_author_delete_withbooks.png)
```
